beacon-chain/rpc/beaconv1/state_test.go

```diff
@@ -80,188 +72,26 @@ func TestGetGenesis(t *testing.T) {
 }
 
 func TestGetStateRoot(t *testing.T) {
-	db := testDB.SetupDB(t)
 	ctx := context.Background()
+	fakeState, err := sharedtestutil.NewBeaconState()
+	require.NoError(t, err)
+	stateRoot, err := fakeState.HashTreeRoot(ctx)
+	require.NoError(t, err)
+	server := &Server{
+		StateFetcher: &testutil.MockFetcher{
+			BeaconStateRoot: stateRoot[:],
+		},
+	}
 
-	t.Run("Head", func(t *testing.T) {
```

Comment:
No need to rewrite all these tests every time now that we use the state fetcher interface. We can just use a mock and test one simple scenario.